### PR TITLE
Use FETS in cdn worker instead of itty-router

### PR DESCRIPTION
**Pull request by ardatan**
image

### CODE REVIEW

File: packages/services/cdn-worker/src/artifact-handler.ts [hunk 1]
```diff
@@ -1,27 +1,24 @@
...
     { type: 'notModified' } | { type: 'notFound' } | { type: 'redirect'; location: string }
   >;
   isKeyValid: KeyValidator;
   analytics?: Analytics;
   fallback?: (
-    request: Request,
+    request: Request | RouterRequest,
     params: { targetId: string; artifactType: string },
-  ) => Promise<Response | undefined>;
+  ) => Promise<any>;
```
Comment (n1ru4l) [inline packages/services/cdn-worker/src/artifact-handler.ts:21]: why any?

File: packages/services/cdn-worker/src/artifact-handler.ts [hunk 2]
```diff
@@ -1,27 +1,24 @@
...
 import type { KeyValidator } from './key-validation';
 
-const { Response } = createFetch({ useNodeFetch: true });
-
 type ArtifactRequestHandler = {
   getArtifactAction: (
     targetId: string,
     artifactType: ArtifactsType,
-    eTag: string | null,
+    eTag?: string | null,
```
Comment (n1ru4l) [inline packages/services/cdn-worker/src/artifact-handler.ts:12]:
```suggestion
    eTag: string | null,
```

Please don't alter my code for no reason 😁

Comment (ardatan) [reply packages/services/cdn-worker/src/artifact-handler.ts:12]: It was an undefined, null conflict. Fixing it in FETS.

File: packages/services/cdn-worker/src/artifact-handler.ts (hunk 1 above)
Comment (ardatan) [reply packages/services/cdn-worker/src/artifact-handler.ts:21]: Somehow default `Response` and `TypedResponse` were conflicting. Adding a workaround to make it pass for now.

**Comment by n1ru4l** [comment]
closing this for now because of staleness and will re-consider it later on.

**Pull request closed without merging**